Sources/DashX/DashXClient.swift

```diff
@@ -650,4 +650,46 @@ public class DashXClient {
             trackNotification(id, event, ISO8601DateFormatter.timeStamp)
         }
     }
+    
+    // MARK: - Create Notification
+    
+    func createNotification(id: String, content: UNMutableNotificationContent) {
```

Comment:
Why can't we create this method in Sources/DashX/DashXAppDelegate.swift. We don't want to expose this as it is

Reply:
made as a private method in the extension of DashX App delegate.